lib/api_repo.js

```diff
@@ -204,6 +204,13 @@ ApiRepo.prototype.buildPackages =
               'packageInfo': that.packageInfo,
               'generated': generated
             }, templateInfo[l]);
+            if (l == 'nodejs' && !that.opts.nodejsUsePbjs) {
+              opts.templates = ['service.js.mustache'];
```

Comment:
Modified the code.
This opts are 'merged' (https://lodash.com/docs#merge) into the default configurations, and I was thinking the arrays are concatenated through merge -- which is wrong. Removed this line, and this code (adding service.js.mustache) is moved to packager.js